garnett/version.py

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2019 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+
+
+__version__ = "1.3.0"
```

Comment:
```suggestion
__version__ = "0.6.1"
```
[![signac](https://docs.signac.io/en/latest/_images/signac.io-dark.svg)](https://signac.io)